webviz_subsurface/plugins/_reservoir_simulation_timeseries.py

```diff
@@ -74,9 +87,7 @@ class ReservoirSimulationTimeSeries(WebvizPluginABC):
 * **`obsfile`**: File with observations to plot together with the relevant time series. \
 (absolute path or relative to config file).
 * **`options`:** Options to initialize plots with:
-    * `vector1` : First vector to display
-    * `vector2` : Second vector to display
-    * `vector3` : Third vector to display
+    * `vectors` : List of vectors to display (max 3 vectors, additional ones are ignored)
```

Comment:
is this breaking change really necessary? Easy to wrap to existing `vector1`, `vector2`, and `vector3` into a list in the code instead (checking if they are defined of course). I agree that it is cleaner, and probably better, but I think it isn't worth it to break existing setups for this.

Reply:
The existing setups are still working and we are showing a deprecation warning. I just wanted to remove it from the docs since it is going to be deprecated, i.e. users new to this plugin should rather stick to the new way of defining vectors. Does that make sense?

Reply:
I see that it is deprecated, I just don't think it is worth it to deprecate it (especially considering we are not making other changes on the input side as far as I know). 
I don't see the point of adding the noise with a deprecation warning (and thus possible questions from users). It is slightly cleaner from the code side of course, but don't underestimate the extra issues you introduce on the user side due to the bunch of existing setups. 

We will probably change other parts of the input for this plugin at a later stage, and considering users don't like to do more changes / updates to their setups than absolutely necessary, I suggest that we delay this deprecation to a time when we make a larger change to the plugin's input.

Reply:
@asnyv I see your point. Well, I am happy with either solution (we would just need to remove the deprecation warning and add back the old doc lines). Your call. What do you think, @anders-kiaer?

Reply:
Lets revert to current input (`vector{1,2,3}`). I like the suggestion to change to list as input, so created #677 and we can take that one when refactoring the `ReservoirSimulationTimeSeries` plugin.